draft-ietf-tls-tls13.md

```diff
@@ -1048,15 +1048,15 @@ as with a 1-RTT handshake with PSK resumption.
          + key_share*
          + psk_key_exchange_modes
          + pre_shared_key
-         (Application Data*)
-         (end_of_early_data)     -------->
+         (Application Data*)     -------->
                                                          ServerHello
```

Comment:
I thought I had seen ekr move the arrows onto their own line in order to make subsequent diffs less noisy, but it probably doesn't matter at this point.